hazelcast/src/main/java/com/hazelcast/spi/impl/eventservice/impl/EventServiceImpl.java

```diff
@@ -677,18 +676,11 @@ public void accept(Packet packet) {
 
     @Override
     public OnJoinRegistrationOperation getPreJoinOperation() {
-        // pre-join operations are only sent by master member
+        // EventService's pre-join operations are only sent by master member
+        // also EventService's pre-join op is sent from joining member to master in JoinRequest
         return getOnJoinRegistrationOperation();
     }
 
-    @Override
-    public Operation getPostJoinOperation() {
```

Comment:
Maybe it can be easier to see the issue if we keep this method and return null by explaining the problem as a comment inside method.